docs/contributing/database-migrations/edd.mdx

```diff
@@ -265,8 +269,7 @@ will not be run until the start of the next deploy when they are moved into `DbS
 After this deploy, to prep for the next release, all migrations in `DbScripts_transition` are moved
 to `DbScripts` and then all migrations in `DbScripts_finalization` are moved to `DbScripts`,
 conserving their execution order for a clean install. For the current branching strategy, PRs will
-be open against `main` when `rc` is cut to prep for this release. This PR automation will also
-handle renaming the migration file and updating any reference of `[dbo_finalization]` to `[dbo]`.
```

Comment:
Needed. Unless you want to remove all references to _finalization and document the manual process instead.